src/components/Table/Table.js

```diff
@@ -0,0 +1,122 @@
+import _ from "lodash";
+import { MDCRipple } from "@material/ripple";
+
+import tableTemplate from "./Table.html";
+import { DotsMenuComponent } from "../DotsMenu/DotsMenu";
+
+export class TableComponent {
+  constructor(mountPoint, props = {}) {
+    this.mountPoint = mountPoint;
+    this.props = props;
+    this.state = {
+      data: this.props.data,
+      initialData: Array.from(this.props.data),
+      orderTypes: ["initial", "asc", "desc"],
+      currentOrderTypeIndex: 1,
+      columnName: ""
+    };
+
+    this.handleOrderClick = this.handleOrderClick.bind(this);
+  }
+
+  getColumnName(target) {
+    if (target.tagName === "BUTTON") {
+      return target.firstElementChild.attributes.data.value;
+    }
+    if (target.tagName === "svg") {
+      return target.attributes.data.value;
+    }
+    return null;
+  }
+
+  handleOrderClick(e) {
+    const { data, initialData, orderTypes, currentOrderTypeIndex } = this.state;
+
+    this.state.columnName = this.getColumnName(e.target);
+    if (!this.state.columnName) {
+      return;
+    }
+
+    if (currentOrderTypeIndex === 0) {
```

Comment:
Should we use constants for order type indexes instead of just numbers like 0,1,2?

Reply:
Nope. Its logic requires array of constants and variable which stands for current index of order type. There are three types so I picked this way. Check out changeCurrentOrderTypeIndex() function which makes the core logic.

Reply:
done